ledger/ledger-api-auth/src/main/scala/com/digitalasset/ledger/api/auth/AuthServiceStatic.scala

```diff
@@ -12,7 +12,7 @@ import io.grpc.Metadata
   *
   * Note: This AuthService is meant to be used for testing purposes only.
   */
-class AuthServiceStatic(claims: PartialFunction[String, Claims]) extends AuthService {
+final class AuthServiceStatic(claims: PartialFunction[String, Claims]) extends AuthService {
```

Comment:
Out of curiosity, what is the reason for adding `final` to all classes in this PR?